src/tests/dynamo0p3_transformations_test.py

```diff
@@ -3350,3 +3369,156 @@ def test_list_multiple_reductions():
         arg.descriptor._access = "gh_sum"
         result = omp_loop_directive._reduction_string()
         assert ", reduction(+:f2), reduction(+:asum)" in result
+
+
+def test_move_name():
+    ''' Test the name property of the MoveTrans class '''
+    move_trans = MoveTrans()
+    name = move_trans.name
+    assert name == "Move"
+
+
+def test_move_str():
+    ''' Test the str method of the MoveTrans class '''
+    move_trans = MoveTrans()
+    name = str(move_trans)
+    assert name == "Move a node to a different location"
+
+
+def test_move_valid_node():
+    '''Test that MoveTrans raises an exception if an invalid node
+    argument is passed'''
+    _, info = parse(os.path.join(BASE_PATH,
+                                 "4.2_multikernel_invokes.f90"),
+                    api=TEST_API)
+    psy = PSyFactory(TEST_API).create(info)
+    invoke = psy.invokes.invoke_list[0]
+    schedule = invoke.schedule
+    move_trans = MoveTrans()
+    with pytest.raises(TransformationError) as excinfo:
+        move_trans.apply(None, schedule.children[0])
+    assert ("In the Move transformation apply method the "
```

Comment:
We should also check that something sensible happens when the second argument to the `apply()` method is not a `Node`.